src/main/java/com/aws/greengrass/clientdevices/auth/session/SessionImpl.java

```diff
@@ -1,48 +1,78 @@
-/*
+ /*
  * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
  * SPDX-License-Identifier: Apache-2.0
  */
 
-package com.aws.greengrass.clientdevices.auth.session;
+ package com.aws.greengrass.clientdevices.auth.session;
 
-import com.aws.greengrass.clientdevices.auth.session.attribute.AttributeProvider;
-import com.aws.greengrass.clientdevices.auth.session.attribute.DeviceAttribute;
+ import com.aws.greengrass.clientdevices.auth.session.attribute.AttributeProvider;
+ import com.aws.greengrass.clientdevices.auth.session.attribute.DeviceAttribute;
+ 
+ import java.util.concurrent.ConcurrentHashMap;
```

Comment:
looks like most lines have an extra space at the start, let's fix the formatting so the diff shows nicely in github